packages/valory/skills/termination_abci/rounds.py

```diff
@@ -179,6 +179,7 @@ class PostTerminationTxAbciApp(AbciApp[Event]):
     # the following is not needed, it is added to satisfy the round check
     # the TerminationRound when run it terminates the agent, so nothing can come after it
     transition_function = {TerminationRound: {Event.TERMINATE: TerminationRound}}
+    cross_period_persisted_keys = ["safe_contract_address"]
```

Comment:
added `cross_period_persisted_keys` to the `PostTerminationTxAbciApp`. It will exist on the `TerminationAbciApp` as well after chaining
https://github.com/valory-xyz/open-autonomy/blob/0db354fff79dffbaeb03c1c11bbda69b8d9c1a07/packages/valory/skills/termination_abci/rounds.py#L182-L199